YAFCmodel/Model/ProductionTable.cs

```diff
@@ -228,6 +229,40 @@ private void CalculateFlow(RecipeRow include)
             flow = flowArr;
         }
 
+        public void RemoveUnusedRecipes(bool unpackToo)
```

Comment:
I tried spamming calls to `RecordUndo` in this method, but that sent the undo system into a stack overflow loop, both with and without unpacking.